tests/unit/lib/Any_unittest.cc

```diff
@@ -691,3 +691,25 @@ TEST(Any, lazy_cloneable_visitable)
 	EXPECT_FALSE(any.visitable());
 	EXPECT_THROW(any.visit(nullptr), std::logic_error);
 }
+
+TEST(Any, compare_string_vectors)
+{
+	std::string str("some string");
+	std::vector<std::string> lhs{str};
+	std::vector<std::string> rhs{str};
+	Any any_lhs = make_any(lhs);
+	Any any_rhs = make_any(rhs);
+	EXPECT_EQ(any_lhs, any_rhs);
+}
+
+TEST(Any, compare_object_vectors)
+{
+	auto lhs_obj = std::make_shared<Object>();
+	auto rhs_obj = std::make_shared<Object>();
+	EXPECT_TRUE(lhs_obj->equals(rhs_obj.get()));
+	std::vector<CSGObject*> lhs{lhs_obj.get()};
+	std::vector<CSGObject*> rhs{rhs_obj.get()};
+	Any any_lhs = make_any(lhs);
+	Any any_rhs = make_any(rhs);
+	EXPECT_EQ(any_lhs, any_rhs);
```

Comment:
wait, did we want == operator for Any? Maybe I forgot this .... ;)
I think we can merge and then I add a vector parameter to our big shogun class test